app/views/shared/eligible_region_content_components/_professional_recognition.html.erb

```diff
@@ -28,6 +27,13 @@
   <%= render "shared/teaching_authority_contact_information", region: %>
```

Comment:
I just spotted this partial is already rendering `teaching_authority_other` so we need to take it out of there, and replace it with the `teaching_authority_status_information` or `teaching_authority_sanction_information`.